src/components/currentNewsHighlights/index.tsx

```diff
@@ -3,8 +3,14 @@ import CurrentNewsCard from './currentNewsCard';
 
 const CurrentNewsHighlights = ({ currentNews }) => (
   <div className="flex flex-row space-x-4 md:w-full my-8">
-    {currentNews.map((item) => (
-      <CurrentNewsCard title={item.title} ingress={item.ingress} date={item.date} buttonText={item.buttonText} />
+    {currentNews.map((item, index: number) => (
+      <CurrentNewsCard
+        key={index}
```

Comment:
Index is a bad value for key, so if there's something else that can be used (like id or title) use that.